firebase_admin/multi_factor_config_mgt.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Firebase multifactor configuration management module.
+
+This module contains functions for managing various multifactor configurations at
+the project and tenant level.
+"""
+
+
+class TotpProviderConfig:
```

Comment:
Can we make this a child class of ProviderConfig? That way ProviderConfig only contains state as a member/property and any future providers can be child classes too. WDYT?

Reply:
The current backend and node SDK have structured the implementation of the class separately. However, I agree that it makes more sense to have it as a nested structure rather than an independent class. This could be added as a feature request to the project backlog for further considerations maybe. Thoughts?

Reply:
I think while this PR is still in the works, it is easier to add now than later.We can wait for the other reviewers chime in too.